.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": false
```

Comment:
@ralfhandl, editor settings probably weren't intended to be introduced to the codebase

Reply:
@OAI/tsc Opinions?

Add `.vscode` to `.gitignore`, or rather set "format on save" to true and encourage/enforce Prettier for code and markdown?

Reply:
I would generally prefer not to have editor-specific directives in .gitignore.  People use a variety of editors, and treating anything like this as if it enforces some sort of policy is at best misleading.  If you want to enforce something on commit, use a commit hook.

Reply:
No traces of VS Code now